pkg/machines/scripts/create_machine.sh

```diff
@@ -23,8 +23,7 @@ handleFailure(){
 if [ "$SOURCE_TYPE" = "disk_image" ]; then
     DISK_OPTIONS="$SOURCE,device=disk"
 else
-    COMPARISON=$(awk 'BEGIN{ print "'$STORAGE_SIZE'"<=0 }')
-    if [ "$COMPARISON" -eq 1 ]; then
+    if [ -z "$STORAGE_SIZE" ] || [ "$STORAGE_SIZE" -le 0 ]; then
```

Comment:
This can be floating point number. I have to revert it to previous state :(

I tried also to use bc but it's not available for the sh.

Reply:
Sorry for the bad advice then! Having floating point numbers is unusual, I didn't realize that.

Reply:
Yes, it's because virt-install accepts only GB as units for new disks size. :/